roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -124,3 +128,11 @@ def parse_version(self, version):
             components[0] = self.openshift_major_release_version[components[0]]
 
         return '.'.join(components[:2])
+
+
+def _v_to_arr(version):
+    return map(int, re.findall(r'\d+', version))
```

Comment:
`map()` in Python 3 does not return a list... that's what causes the Travis error in `_version_lt` when we try to compare the two objects returned by `_v_to_arr`.

Suggestion:

```python
return tuple(int(component) for component in re.findall(r'\d+', version))
```